client/jetpack-onboarding/steps/summary.jsx

```diff
@@ -40,18 +40,24 @@ class JetpackOnboardingSummaryStep extends React.PureComponent {
 
 	renderTodo = () => {
 		const stepsTodo = [
+			SUMMARY_STEPS.JETPACK_CONNECTION,
 			SUMMARY_STEPS.THEME,
 			SUMMARY_STEPS.SITE_ADDRESS,
 			SUMMARY_STEPS.STORE,
 			SUMMARY_STEPS.BLOG,
 		];
-
-		// TODO: adapt when we have more info + it will differ for different steps
-		const siteRedirectHref = '#';
+		const stepLinks = [
+			'/jetpack/connect?url=' + this.props.siteUrl,
+			// TODO: update the following with relevant links
+			'#',
+			'#',
+			'#',
+			'#',
+		];
```

Comment:
I think `stepLinks` should be tied stronger to `stepsTodo`. Maybe just a `steps` array, with the consts as names, and objects as values (that contain a `label` and a `url` property)?

```js
const steps = {
	SUMMARY_STEPS.JETPACK_CONNECTION: {
		label: this.props.translate( 'Jetpack Connection' ),
		url: '/jetpack/connect?url=' + this.props.siteUrl,
	},
	SUMMARY_STEPS.JETPACK_THEME: { /*...*/ },
	/* etc */
}
```

Reply:
Oh, and I think we should change the wording here from a noun ('Jetpack Connection') to a verb that sounds more like an action (something like 'Connect your site to WordPress.com') (much like 'Choose a Theme')